src/components/learnDetail/record/RecordStatusBar.tsx

```diff
@@ -0,0 +1,236 @@
+    formData.append('scriptId', scriptId.toString());
+    formData.append('endtime', duration.toString());
+    formData.append('date', getDate());
+    setRecordFormData(formData);
+  };
+
+  const getDate = () => {
+    const TIME_ZONE = 3240 * 10000;
+    const date = new Date();
+    const dateInForm = new Date(+date + TIME_ZONE).toISOString().split('T')[0];
+    const time = date.toTimeString().split(' ')[0];
+    return dateInForm + ' ' + time;
+  };
+
+  const handleTimerSecond = (second: number) => {
+    if (second === 60) {
+      setSeconds(0);
+      setMinutes((prev) => prev + 1);
+    }
+    return second;
+  };
+
+  const twoDigitsNumber = (num: number) => {
+    return String(num).padStart(2, '0');
+  };
+
+  return (
+    <>
+      <StRecordStatusBar
+        onClick={(e) => {
+          e.stopPropagation();
+          !isLoggedIn && setIsLoginModalOpen(true);
+        }}>
+        <StRecordStatus isRecording={isRecording}>
+          {isRecording ? (
+            <ImageDiv src={icRecordMicActive} className="icRecordMic" layout="fill" alt="" />
+          ) : (
+            <ImageDiv src={icRecordMicDefault} className="icRecordMic" layout="fill" alt="" />
+          )}
+
+          <RecordTime isRecording={isRecording}>
+            {twoDigitsNumber(minutes)}:{twoDigitsNumber(handleTimerSecond(seconds))}
+          </RecordTime>
+          {isRecording ? (
+            <ImageDiv src={icRecordStop} className="icRecordStop" layout="fill" alt="녹음 중지" onClick={stopRecord} />
+          ) : (
+            <ImageDiv
+              src={icRecordStart}
+              className="icRecordStart"
+              layout="fill"
+              alt="녹음 시작"
+              onClick={() => {
+                isLoggedIn && startRecord();
+              }}
+            />
+          )}
+        </StRecordStatus>
+        {isSaved && (
+          <>
+            <ImageDiv src={icRecordSaveToast} className="icRecordSaveToast" layout="fill" alt="" />
+            <p>저장되었습니다.</p>
+          </>
+        )}
+      </StRecordStatusBar>
+      {isLoginModalOpen && <LoginModal closeModal={() => setIsLoginModalOpen(false)} />}
+    </>
+  );
+}
+
+export default RecordStatusBar;
+
+const StRecordStatusBar = styled.div`
+  width: 14.1rem;
+  height: 4.8rem;
+  margin-right: 54.9rem;
+
+  p {
+    position: absolute;
+    bottom: 7rem;
+    left: 7rem;
+    color: ${COLOR.WHITE};
+    ${FONT_STYLES.SB_15_CAPTION};
+  }
+
+  .icRecordSaveToast {
+    position: absolute;
+    bottom: 5.2rem;
+    left: 6rem;
+    width: 11.4rem;
+    height: 4.9rem;
+  }
+`;
+
+const StRecordStatus = styled.div<{ isRecording: boolean }>`
+  display: flex;
+  align-items: center;
+  width: 14.1rem;
+  height: 4.8rem;
+  margin-right: 54.9rem;
+  border-radius: 5rem;
+
+  ${({ isRecording }) =>
+    isRecording
+      ? css`
+          border: 0.2rem solid ${COLOR.MAIN_BLUE};
+        `
+      : css`
+          border: 0.2rem solid ${COLOR.GRAY_5};
+        `}
```

Comment:
```suggestion
  border: 0.2rem solid ${({ isRecording }) => (isRecording ? COLOR.MAIN_BLUE : COLOR.GRAY_5)};
```
이렇게 바꿔보는 것은 어떨까요?
너무 길다 싶으면 `border-color`만 따로 빼서 작성하는 것도 좋을 것 같아요.

밑에 RecordTime의 color도 마찬가지입니다!

Reply:
덕분에 코드 줄였어용! 짱짱